mne/epochs.py

```diff
@@ -2132,9 +2132,18 @@ def _read_one_epoch_file(f, tree, fname, preload):
         if len(processed) == 0:
             raise ValueError('Could not find processed data')
 
-        epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
+        if fix_tags:
+            epochs_node = dir_tree_find(tree, 122)
```

Comment:
and you might as well use the actual constant we were accidentally saving it under, too. You could add a comment like "before version 0.11 we errantly saved with this tag instead of an MNE tag".